src/status_im/chat/models/images.cljs

```diff
@@ -21,25 +21,20 @@
                          :path   temp-image-url}))
       (.fetch "GET" base64-uri)
       (.then #(on-success (.path %)))
-      (.catch #(log/error "could not save image"))))
+      (.catch #(log/error (str "could not download image" %)))))
 
-(defn save-to-gallery [path] (.save CameraRoll path))
+(defn save-image
+  [path]
+  (-> (.save CameraRoll path)
+      (.then #(fs/unlink path))
```

Comment:
should we leave a comment explain why it needs to be unlinked here or it's straightforward?

Reply:
IMO this `save-image` functionality should be isolated inside `react-native.cameraroll`. That way, we make an even stronger stance via code that unlink must be called after `save` is successful.